.github/workflows/server-release.yml

```diff
@@ -7,21 +7,55 @@ on:
 jobs:
   release:
     name: release ${{ matrix.target }}
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         include:
           - target: x86_64-pc-windows-gnu
-            archive: zip
+            os: windows-latest
+
           - target: x86_64-unknown-linux-musl
-            archive: tar.gz tar.xz
+            os: ubuntu-latest
+
+          - target: x86_64-apple-darwin
+            os: macos-latest
+
+          - target: aarch64-apple-darwin
+            os: macos-latest
     steps:
       - uses: actions/checkout@master
-      - name: Compile and release
-        uses: rust-build/rust-build.action@latest
+
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+          profile: minimal
+          target: ${{ matrix.target }}
+
+      - name: Install musl for x86_64-unknown-linux-musl
+        if: ${{ matrix.target == 'x86_64-unknown-linux-musl' }}
+        run: sudo apt-get install -y musl-dev musl-tools
+
+      - name: Compile server
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --target ${{ matrix.target }} --release --manifest-path server/svix-server/Cargo.toml
+
+      - name: Set SDKROOT for MacOS
```

Comment:
Can these two macos conditions be merged and just check the is name? They look essentially identical, and I wonder whether it makes sense.